remme/shared/messaging.py

```diff
@@ -0,0 +1,302 @@
+# Copyright 2018 REMME
```

Comment:
Classes in this file should also be documented.

Reply:
U mean in `.rst`

Reply:
in docstrings

Reply:
Description already there.